cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -759,6 +759,8 @@ def _validate_colors_in_uns_dict(self, uns_dict: dict) -> None:
                 category_mapping[column_name] = column.nunique()
 
         for key, value in uns_dict.items():
+            if len(value) == 0:
+                self.errors.append(f"uns['{key}'] cannot be an empty value.")
```

Comment:
I think this should be moved down to be integrated with the `if len(value) < obs_unique_values` statement on ln 801, because this is a higher-order error, and so it should strictly supercede the error condition on 801. Integrate as an `if len(value) == 0... elif len(value) < obs_unique_values`

Reply:
Although admittedly that is not a colors-related check... so it would be strange to have it as part of the colors logic loop.

Maybe instead change ln 801 to be `if len(value) > 0 and len(value) < obs_unique_values`

Add a comment to indicate the zero condition is checked independently earlier in the function.

Reply:
Yeah we have to do it that way because we need to check all values for zero length, not just the ones that have a `_colors` key, so disregard my first comment in this thread